synapseclient/table.py

```diff
@@ -285,6 +285,7 @@
  - :py:meth:`synapseclient.Synapse.store`
  - :py:meth:`synapseclient.Synapse.delete`
 """
+from __future__ import annotations
```

Comment:
Is this still needed?

Reply:
I don't think so. I just removed it. Thanks for catching.